tests/test_output_formats.py

```diff
@@ -0,0 +1,127 @@
+import json
+from collections import OrderedDict
+
+from code42cli.output_formats import output_format
+from code42cli.output_formats import to_csv
+from code42cli.output_formats import to_formatted_json
+from code42cli.output_formats import to_json
+from code42cli.output_formats import to_table
+
+
+TEST_DATA = [
+    {
+        "type$": "RULE_METADATA",
+        "modifiedBy": "test.user+partners@code42.com",
+        "modifiedAt": "2020-06-22T16:26:16.3875180Z",
+        "name": "outside td",
+        "description": "",
+        "severity": "HIGH",
+        "isSystem": False,
+        "isEnabled": True,
+        "ruleSource": "Alerting",
+        "tenantId": "1d71796f-af5b-4231-9d8e-df6434da4663",
+        "observerRuleId": "d12d54f0-5160-47a8-a48f-7d5fa5b051c5",
+        "type": "FED_CLOUD_SHARE_PERMISSIONS",
+        "id": "5157f1df-cb3e-4755-92a2-0f42c7841020",
+        "createdBy": "test.user+partners@code42.com",
+        "createdAt": "2020-06-22T16:26:16.3875180Z",
+    },
+    {
+        "type$": "RULE_METADATA",
+        "modifiedBy": "testuser@code42.com",
+        "modifiedAt": "2020-07-16T08:09:44.4345110Z",
+        "name": "Test different filters",
+        "description": "Test different filters",
+        "severity": "MEDIUM",
+        "isSystem": False,
+        "isEnabled": True,
+        "ruleSource": "Alerting",
+        "tenantId": "1d71796f-af5b-4231-9d8e-df6434da4663",
+        "observerRuleId": "8b393324-c34c-44ac-9f79-4313601dd859",
+        "type": "FED_ENDPOINT_EXFILTRATION",
+        "id": "88354829-0958-4d60-a20d-69a53cf603b6",
+        "createdBy": "test.user+partners@code42.com",
+        "createdAt": "2020-05-20T11:56:41.2324240Z",
+    },
+    {
+        "type$": "RULE_METADATA",
+        "modifiedBy": "testuser@code42.com",
+        "modifiedAt": "2020-05-28T16:19:19.5250970Z",
+        "name": "Test Alerts using CLI",
+        "description": "spatel",
+        "severity": "HIGH",
+        "isSystem": False,
+        "isEnabled": True,
+        "ruleSource": "Alerting",
+        "tenantId": "1d71796f-af5b-4231-9d8e-df6434da4663",
+        "observerRuleId": "5eabed1d-a406-4dfc-af81-f7485ee09b19",
+        "type": "FED_ENDPOINT_EXFILTRATION",
+        "id": "b2cb33e6-6683-4822-be1d-8de5ef87728e",
+        "createdBy": "testuser@code42.com",
+        "createdAt": "2020-05-18T11:47:16.6109560Z",
+    },
+]
+
+TEST_HEADER = OrderedDict()
+TEST_HEADER["observerRuleId"] = "RuleId"
+TEST_HEADER["name"] = "Name"
+TEST_HEADER["severity"] = "Severity"
+TEST_HEADER["type"] = "Type"
+TEST_HEADER["ruleSource"] = "Source"
+TEST_HEADER["isEnabled"] = "Enabled"
+
+TABLE_OUTPUT = """RuleId                                 Name                     Severity   Type                          Source     Enabled   
```

Comment:
The style guide is complaining that there are trailing spaces at the end of line. Tried adding `"...\n"` approach too but then tests weren't happy. Need suggestions to fix this silly thing or shall I just change the function to not add any padding after the last column?

Reply:
Fixed now.